app/code/Magento/Sales/Block/Adminhtml/Order/View/Tab/History.php

```diff
@@ -218,6 +246,35 @@ protected function _prepareHistoryItem($label, $notified, $created, $comment = '
         return ['title' => $label, 'notified' => $notified, 'comment' => $comment, 'created_at' => $created];
     }
 
+    /**
+     * Map history items as array with id and type
+     *
+     * @param int $id
+     * @param string $label
+     * @param bool $notified
+     * @param \DateTimeInterface $created
+     * @param string $comment
+     * @param int $type
+     * @return array
+     */
+    private function _prepareHistoryItemWithId(
```

Comment:
Please remove the underscore from the method name, it was an old practice from Magento 1, the new code shouldn't contain it
```suggestion
    private function prepareHistoryItemWithId(
```

Reply:
@ihor-sviziev I have updated this function name and addressed the other un-related files that creeped into this pull request.  Shame on me for not following git best-practices...  Sorry this got a bit messy.  I think this is all straightened out now.